x-pack/plugins/alerting/server/lib/get_execution_log_aggregation.test.ts

```diff
@@ -2233,7 +2285,8 @@ describe('formatExecutionKPIAggBuckets', () => {
                     sum_other_doc_count: 0,
                     buckets: [
                       {
-                        key: 'success',
+                        key: ['success', 'success'],
+                        key_as_string: 'success|success',
```

Comment:
Should we add a test for when the `key_as_string` is `success|warning` or `|success` to test that those are parsed correctly? Alternatively, you could extract that logic out into a smaller function and just unit test that function.

Reply:
Sure, I can do that!

Reply:
Resolved in 2e3408c4a60c333fd215d30cb9ad96973b411529